src/renderer/services/file-management-system/index.ts

```diff
@@ -137,15 +136,26 @@ export default class FileManagementSystem {
           lastModifiedInMS: fileLastModifiedInMs,
         },
       });
-
-      // Wait for upload
-      await this.uploadInChunks({
-        fssUploadId: registration.uploadId,
-        source: source,
-        chunkSize: registration.chunkSize,
-        user: upload.user,
-        onProgress: (bytesUploaded) => onProgress({ bytesUploaded, totalBytes: fileSize })
-      });
+      if (registration.chunkStatuses && registration.chunkStatuses[0]){
```

Comment:
Does this need to check that the status (at 0) is successful or any specific status or is it just existing good enough?

Reply:
Just existing should be good enough; the retry/resume code should handle it.